FHIR-Converter/client/src/view/common/registry/pull-image.ts

```diff
@@ -43,7 +43,7 @@ export async function pullImage(imageReference, refineOutput= false) {
 		
 		if (refineOutput) {
 			// Fix the wrong message from engine temporarily
-			output = output.replace('pulled templates to', 'pushed sample data to');
+			output = output.replace('pulled templates to', 'pulled sample data to');
```

Comment:
Could we push the process for pullbar and output information outside this function? maybe in pull-official-templates.ts  and pull-sample-data.ts ? Which can simplify function's parameters.

Reply:
This change only temporarily processes the output of engine and will be removed later, so try to change other places as little as possible. In the next version of engine, I think we can change the message of engine to the more general message like "pulled files to".